examples/fromjson/main.go

```diff
@@ -1,3 +1,6 @@
+//go:build mage
```

Comment:
why that ?

Reply:
There was originally a "mage", but I accidentally deleted it and committed it.

Reply:
I'm sorry, but you don't use `mage` now?

Reply:
not for such file (mage is used to control build basically)

Reply:
@jbguerraz 
i'm sorry. i deleted mage
I'd like a code review